sessionctx/variable/session.go

```diff
@@ -315,6 +323,8 @@ func NewSessionVars() *SessionVars {
 		RetryLimit:                DefTiDBRetryLimit,
 		DisableTxnAutoRetry:       DefTiDBDisableTxnAutoRetry,
 		DDLReorgPriority:          kv.PriorityLow,
+		EnableRadixJoin:           false,
+		L2CacheSize:               cpuid.CPU.Cache.L2,
```

Comment:
I am not familiar with golang package management, where does `cpuid.CPU.Cache.L2` variable initialized? we don't need to call a init function in package `cpuid`to set up this variable?

Reply:
We do not need to init it since it has been inited in the third package.
Refer: https://github.com/klauspost/cpuid/blob/master/cpuid.go#L180-L205

Reply:
Got it, thanks